fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/StreamUtils.java

```diff
@@ -20,16 +20,40 @@
 import static java.util.Spliterator.IMMUTABLE;
 import static java.util.Spliterators.spliteratorUnknownSize;
 import static java.util.stream.StreamSupport.stream;
+import static org.slf4j.LoggerFactory.getLogger;
 
 import java.util.Iterator;
 import java.util.stream.Stream;
 
+import org.slf4j.Logger;
+
 /**
  * @author acoburn
  * @since February 10, 2016
  */
 public class StreamUtils {
 
+    private static final Logger LOGGER = getLogger(StreamUtils.class);
+
+    private static Boolean enableParallel = false;
+
+    private static final String FCREPO_ENABLE_PARALLEL_STREAMING_BY_DEFAULT_KEY =
```

Comment:
Once the PR is in the codebase, we should also update the documentation with this configuration element.
https://wiki.duraspace.org/display/FEDORA4x/Configuration+Options+Inventory